AmethystAPI/src/minecraft/src/common/world/level/storage/GameRules.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <amethyst/Memory.hpp>
+
+class GameRule {
+public:
+  /* this + 0 */ std::byte padding0[1];
```

Comment:
^ While this line is accidentally right in this scenario, you should make it explicitly a padding of 8 bytes, otherwise you gamble with the c++ compile whether it decides to align stuff in your favour.

Reply:
Whoops, this was for debugging and it seems that I forgot to remove it. My laptop is almost dead so I can't fix it and verify it works now, so you can just leave the PR as-is and I'll notify you when it's fixed.